src/components/PanelHeaderSubmit/PanelHeaderSubmit.tsx

```diff
@@ -22,7 +22,7 @@ export const PanelHeaderSubmit = ({
       primary
       {...restProps}
     >
-      {platform === ANDROID || platform === VKCOM ? (
+      {platform === Platform.ANDROID || platform === Platform.VKCOM ? (
```

Comment:
```suggestion
      {platform !== Platform.IOS ? (
```
(ну или `platform === Platform.IOS` и поменять местами выражения)